### PR TITLE
FiftyOne Collaboration Updates

### PR DESCRIPTION
**Pull request by ehofesmann**
COCO is about to be easier to load, visualize and evaluate than ever! :smiley: 

This PR updates the Home, Download, and Detection Evaluation pages demonstrating how to use the COCO integration in [FiftyOne](https://fiftyone.ai), an open-source dataset visualization and model analysis tool, to download/visualize subsets of the dataset and evaluate models on COCO. 

The new release of FiftyOne with this integration is scheduled to be announced on Thursday, June 24th, please wait to update the website until then.

Below are example screenshots of the updated webpages.

## Home
image

## Download
image

## Evaluate Detections
image

### CODE REVIEW

File: dataset/home.htm [hunk 1]
```diff
@@ -4,6 +4,7 @@ <h1>News</h1>
   <li>We are pleased to announce the COCO 2020 <a class="fontBoldish" href="#detection-2020">Detection</a>, <a class="fontBoldish" href="#keypoints-2020">Keypoint</a>, <a class="fontBoldish" href="#panoptic-2020">Panoptic</a>, and <a class="fontBoldish" href="#densepose-2020">DensePose</a> Challenges.<br/></li>
   <li class="fontBoldish">The new rules and awards for this year challenges encourage innovative methods.</li>
   <li>Results to be announced at the <a class="fontBoldish" href="workshop/coco-lvis-eccv-2020.html">Joint COCO and LVIS Recognition</a> ECCV workshop.</li>
+  <li>We have partnered with the team behind the open-source tool <a class="fontBoldish" href="https://fiftyone.ai">FiftyOne</a> to make it easier to download, visualize, and evaluate COCO</li>
```
Comment (tylin) [inline dataset/home.htm:7]: FiftyOne provides open-source tool to visualize and facilitate access to COCO data resources and an evaluation tool to breakdown the detection performance on COCO.

File: dataset/download.htm [hunk 2]
```diff
@@ -84,3 +84,48 @@ <h1>3. MASK API</h1>
...
+)
+
+# Visualize the dataset in the FiftyOne App
+session = fiftyone.launch_app(dataset)
+</pre>
+
+<p>
+Once you start training models on COCO, you can use
+<a href="https://cocodataset.org/#detection-eval">FiftyOne's COCO-style evaluation</a>
+to better understand your model performance,
```
Comment (tylin) [inline dataset/download.htm:122]: to understand your model performance,

File: dataset/download.htm [hunk 3]
```diff
@@ -84,3 +84,48 @@ <h1>3. MASK API</h1>
...
+
+<h1>4. FiftyOne</h1>
+<p>We have partnered with the team behind <a href="https://fiftyone.ai">FiftyOne</a>, an
+open-source tool for dataset visualization and model analysis, to make it
+easier to download, visualize, and evaluate COCO.
+</p>
+<p><a href="https://fiftyone.ai"><img class="wide" src="images/fiftyone-example.png"></a></p>
+
+<p>
+Downloading COCO is now as easy as any other dataset in the
```
Comment (tylin) [inline dataset/download.htm:96]: Downloading COCO in the

File: dataset/download.htm [hunk 4]
```diff
@@ -84,3 +84,48 @@ <h1>3. MASK API</h1>
   <div class="jsonkfunc">frBbox</div><div class="jsonvtxt">Convert bounding boxes to encoded masks.</div>
   <div class="jsonkfunc">frPoly</div><div class="jsonvtxt">Convert polygon to encoded mask.</div>
 </div>
+
+<h1>4. FiftyOne</h1>
+<p>We have partnered with the team behind <a href="https://fiftyone.ai">FiftyOne</a>, an
```
Comment (tylin) [inline dataset/download.htm:89]: FiftyOne provides open-source tool to visualize and facilitate access to COCO data resources and an evaluation tool for model analysis on COCO.

File: dataset/detection-eval.htm [hunk 5]
```diff
@@ -91,3 +91,24 @@ <h1>4. Analysis Code</h1>
...
 <p>For a given detector, the code generates a total of 372 plots! There are 80 categories, 12 supercategories, and 1 overall result, for a total of 93 different settings, and the analysis is performed at 4 scales (all, small, medium, large, so 93*4=372 plots). The file naming is [supercategory]-[category]-[size].pdf for the 80*4 per-category results, overall-[supercategory]-[size].pdf for the 12*4 per supercategory results, and overall-all-[size].pdf for the 1*4 overall results. Of all the plots, typically the overall and supercategory results are of the most interest.</p>
 <p><b>Note:</b> <span class="fontMono">analyze()</span> can take significant time to run, please be patient. As such, we typically do not run this code on the evaluation server; you must run the code locally using the validation set. Finally, currently <span class="fontMono">analyze()</span> is only part of the Matlab API; <i>Python code coming soon</i>.</p>
+
+<h1>5. Visualization Code</h1>
+<p>We have partnered with the team behind <a href="https://fiftyone.ai">FiftyOne</a>, an
```
Comment (tylin) [inline dataset/detection-eval.htm:96]: FiftyOne provides open-source tool to visualize and facilitate access to COCO data resources and an evaluation tool for model analysis on COCO.

**Review by tylin — CHANGES_REQUESTED** [review]
Hi Eric, thanks for putting up the update. Here are some changes we'd like to make.

**Comment by ehofesmann** [comment]
> Hi Eric, thanks for putting up the update. Here are some changes we'd like to make.

@tylin Thanks for the feedback! I'll update it right away

**Pull request merged**